geode-core/src/test/resources/org/apache/geode/management/internal/cli/commands/golden-help-offline.properties

```diff
@@ -2572,7 +2572,7 @@ SYNTAX\n\
 \ \ \ \ [--lock-memory(=value)?] [--log-level=value] [--max-connections=value] [--max-heap=value]\n\
 \ \ \ \ [--max-message-count=value] [--max-threads=value] [--mcast-address=value] [--mcast-port=value]\n\
 \ \ \ \ [--memcached-port=value] [--memcached-protocol=value] [--memcached-bind-address=value]\n\
-\ \ \ \ [--compatible-with-redis-port=value] [--compatible-with-redis-bind-address=value] [--compatible-with-redis-password=value]\n\
+\ \ \ \ [--geode-for-redis-port=value] [--geode-for-redis-bind-address=value] [--geode-for-redis-password=value]\n\
```

Comment:
I'm pretty sure this file can just be deleted - I don't see any references to it anywhere. Even before your changes it looks like it is referencing `geode-for-redis-password` which is no longer a valid param (I missed this reference when I removed that parameter).